source/features/show-whitespace.css

```diff
@@ -6,19 +6,11 @@
 }
 
 [data-rgh-whitespace='tab'] {
-	background-image: url('data:image/svg+xml,%3Csvg preserveAspectRatio="xMinYMid meet" viewBox="0 0 12 24" xmlns="http://www.w3.org/2000/svg"%3E%3Cpath d="M9.5 10.44L6.62 8.12L7.32 7.26L12.04 11V11.44L7.28 14.9L6.62 13.9L9.48 11.78H0V10.44H9.5Z" fill="rgba(36,41,46,25%)"/%3E%3C/svg%3E');
+	background-image: url('data:image/svg+xml,%3Csvg preserveAspectRatio="xMinYMid meet" viewBox="0 0 12 24" xmlns="http://www.w3.org/2000/svg"%3E%3Cpath d="M9.5 10.44L6.62 8.12L7.32 7.26L12.04 11V11.44L7.28 14.9L6.62 13.9L9.48 11.78H0V10.44H9.5Z" fill="rgba(128, 128, 128, 50%)"/%3E%3C/svg%3E');
 	background-size: calc(var(--tab-size, 4) * 1ch) 1.25em;
 }
 
 [data-rgh-whitespace='space'] {
-	background-image: url('data:image/svg+xml,%3Csvg preserveAspectRatio="xMinYMid meet" viewBox="0 0 12 24" xmlns="http://www.w3.org/2000/svg"%3E%3Cpath d="M4.5 11C4.5 10.1716 5.17157 9.5 6 9.5C6.82843 9.5 7.5 10.1716 7.5 11C7.5 11.8284 6.82843 12.5 6 12.5C5.17157 12.5 4.5 11.8284 4.5 11Z" fill="rgba(36,41,46,25%)"/%3E%3C/svg%3E');
+	background-image: url('data:image/svg+xml,%3Csvg preserveAspectRatio="xMinYMid meet" viewBox="0 0 12 24" xmlns="http://www.w3.org/2000/svg"%3E%3Cpath d="M4.5 11C4.5 10.1716 5.17157 9.5 6 9.5C6.82843 9.5 7.5 10.1716 7.5 11C7.5 11.8284 6.82843 12.5 6 12.5C5.17157 12.5 4.5 11.8284 4.5 11Z" fill="rgba(128, 128, 128, 50%)"/%3E%3C/svg%3E');
```

Comment:
This is what I had initially suggested, a color that sits right in the middle and works for both. I took these comparative screenshots but I don't remember which one is _before_ and which is the _after_, but they're almost identical anyway

<img width="312" alt="Screen Shot 7" src="https://user-images.githubusercontent.com/1402241/104968112-97248f00-59aa-11eb-9007-2e3bd58bd8a6.png">
<img width="312" alt="Screen Shot 6" src="https://user-images.githubusercontent.com/1402241/104968113-9855bc00-59aa-11eb-8d19-2830ccd17e92.png">
<img width="314" alt="Screen Shot 5" src="https://user-images.githubusercontent.com/1402241/104968114-98ee5280-59aa-11eb-8775-3f095853887f.png">
<img width="308" alt="Screen Shot 4" src="https://user-images.githubusercontent.com/1402241/104968116-9986e900-59aa-11eb-9b4e-2df42dcd9085.png">